### PR TITLE
Add links in README to reference Sonarqube FE/BE

### PR DESCRIPTION
**Pull request by scprek**
## Hey, I just made a Pull Request!

The doc link from Backstage.io plugins sonarqube lead to the Frontend readme https://github.com/backstage/backstage/blob/master/plugins/sonarqube/README.md. Although it's probably common knowledge to always check if there's a backend element of the plugin, adding links to the READMEs help new users.

#### :heavy_check_mark: Checklist

- [ ] Added or updated documentation
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by OrkoHunter — APPROVED** [review]
Thanks!

**Comment by OrkoHunter** [comment]
> Thanks for the contribution! All commits need to be DCO signed before they are reviewed. Please refer to the the [DCO section in CONTRIBUTING.md](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin) or the [DCO](https://github.com/backstage/backstage/runs/10048832441) status for more info.

@scprek Just a tiny change required to amend and sign the commit. :)

**Review by OrkoHunter — DISMISSED** [review]
Needs DCO update.

**Review by Rugvip — CHANGES_REQUESTED** [review]
Thank you! 👍 

We'll need changesets as well for this to get released, see https://github.com/backstage/backstage/pull/15193#issuecomment-1347404674

**Comment by scprek** [comment]
@Rugvip fixed it. I initially signed with my personal user and that wasn't working, so just used my work one.

**Comment by scprek** [comment]
> Thank you! 👍
> 
> We'll need changesets as well for this to get released, see [#15193 (comment)](https://github.com/backstage/backstage/pull/15193#issuecomment-1347404674)

@Rugvip fixed

File: .changeset/witty-phones-chew.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-sonarqube': patch
+'@backstage/plugin-sonarqube-backend': patch
+---
+
+Sonarqube doc change to link FE/BE changes needed
```
Comment (Rugvip) [inline .changeset/witty-phones-chew.md:6]:
```suggestion
Added links to the frontend and backend plugins in the readme.
```

**Review by Rugvip — APPROVED** [review]
Thanks! 👍 

I tweaked it a bit to make it a bit more oriented towards the changelog reader

**Pull request merged**